src/minter-suite-mapping.ts

```diff
@@ -95,6 +95,14 @@ import {
   UnregisteredNFTAddress as MinterHolderV1UnregisteredNFTAddress,
   RemovedHoldersOfProjects as MinterHolderV1RemovedHoldersOfProjects
 } from "../generated/MinterHolderV1/MinterHolderV1";
+import {
+  AllowedHoldersOfProjects as MinterHolderV2AllowedHoldersOfProjects,
+  RegisteredNFTAddress as MinterHolderV2RegisteredNFTAddress,
+  UnregisteredNFTAddress as MinterHolderV2UnregisteredNFTAddress,
+  RemovedHoldersOfProjects as MinterHolderV2RemovedHoldersOfProjects
+} from "../generated/MinterHolderV2/MinterHolderV2";
+import { DelegationRegistryUpdated as MinterHolderV2DelegationRegistryUpdated } from "../generated/MinterHolderV2/IFilteredMinterHolderV1";
+import { DelegationRegistryUpdated as MinterMerkleV3DelegationRegistryUpdated } from "../generated/MinterMerkleV3/IFilteredMinterMerkleV1";
```

Comment:
nit: Since these events come from the minter interfaces instead of the specific version of Merkle/Holder minters, can we omit the version numbers, and instead import the Holder and Merkle events as `MinterHolderDelegationRegistryUpdated` and `MinterMerkleDelegationRegistryUpdated`, respectively?